### ISSUE TITLE
No support for accessory types

### ISSUE THREAD

**Issue by HunterJKirby** [bendodson/homebridge-applescript#2]
I have a thermostat that I can control with AppleScript but would love to make it an actual HomeKit thermostat instead of a switch, fan, or light

**Comment by beeper99** [comment]
Yes I would to change mine to an irrigation system, HomeKit supports this.
